cli/tests/containerize.bats

```diff
@@ -106,7 +106,7 @@ create_and_start_podman_machine() {
   # For some of these calls you need to not only close FD 3, but also FD 4.
   # I have no idea why.
   echo "Creating podman machine" >&3
-  podman machine init -v /tmp:/tmp -v /Users:/Users -v /private:/private flox-containerize-vm 3>&- 4>&-
+  podman machine init -v /private/tmp:/private/tmp -v /Users:/Users -v /private:/private flox-containerize-vm 3>&- 4>&-
```

Comment:
Lol well that ended up being a small fix

Is your understanding that `/tmp` isn't supported but `/private/tmp` is? I couldn't quite tell if `/private/tmp` was supported on purpose or accident
> Regardless, I’d say that sharing /tmp across machines is risky. Frequently enough processes tend to assume (without a very reason) that some names are exclusive to them, and to that machine (consider the hard-coded X11 socket paths, or perhaps something using PID files (with no machine ID) to disambiguate).
>
> I don’t know how much effort it makes sense to spend on supporting this.

https://github.com/containers/podman/issues/18230#issuecomment-1554768539